konflux-ci/argocd/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
This should not be deployed when using the EKS cluster and Terraform take care of deploying ArgoCD.